### ISSUE TITLE
`maha_dist` incompatible with `tbl_df`

### ISSUE THREAD

**Issue by wjschne** [tonyfischetti/assertr#25]
Data frames with a `tbl_df` wrapper (e.g., data imported with the `readr` package) raise an error in the `maha_dist` function.

**Comment by tonyfischetti** [comment]
Good catch! The fix is incredibly easy so you can expect it really soon! 
In the mean time, you can just call `data.frame()` on it.
In the process of investigating it, I discovered that `data.matrix` doesn't convert non-factor strings into numbers like I assumed it did. Thanks again for catching this!
